vspace/src/main/java/edu/asu/diging/vspace/web/staff/StaffSearchSpaceController.java

```diff
@@ -22,29 +26,33 @@ public class StaffSearchSpaceController {
 
     @Autowired
     private IStaffSearchManager staffSearchManager;
-
-    @RequestMapping(value = "/staff/search/space")
```

Comment:
what about the search page that is using this controller?

Reply:
this is not used anywhere, there is a StaffSearch controller that is rendering the spaces, on being searched

Reply:
What about here: https://github.com/diging/virtual-spaces-2.0/blob/89ac3e069050cad0d3f9eb19414b765ca2362211/vspace/src/main/webapp/WEB-INF/views/staff/search/staffSearch.html#L68